include/sys/zio_crypt.h

```diff
@@ -0,0 +1,191 @@
+/*
+ * CDDL HEADER START
+ *
+ * The contents of this file are subject to the terms of the
+ * Common Development and Distribution License (the "License").
```

Comment:
This and any other new files that are not derived from existing files under the CDDL should be modified to state `Common Development and Distribution License, Version 1.0 only` so that Oracle cannot exercise sections 4.1 and 4.2 of the CDDL to put them under an open source incompatible license. There are already a few files in the repository like this, such as `./lib/libspl/xdr.c`.